_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -28,6 +28,26 @@ planning until early May, or make travel plans with refundable fares.
 
 We will update this webpage as information comes to hand.
 
+## Registration
+
+To register for JuMP-dev 2020, [purchase a ticket](https://www.eventbrite.com/e/jump-dev-2020-tickets-99226534235).
+
+Ticket prices are:
```

Comment:
We should add that it will be refunded in case the physical event is cancelled. Or is the virtual event not free either? Anyway, the dinner should be refunded